apps/dav/lib/Search/EventsSearchProvider.php

```diff
@@ -228,4 +271,19 @@ protected function isDayEqual(
 	): bool {
 		return $dtStart->format('Y-m-d') === $dtEnd->format('Y-m-d');
 	}
+
+	public function getSupportedFilters(): array {
+		return [
+			'term',
+			'person',
```

Comment:
You also support them:
```suggestion
			'person',
			'since',
			'until',
```